### ISSUE TITLE
Unable to re-enable built-in extension

### ISSUE THREAD

**Issue by seahindeniz** [microsoft/vscode#102905]
- VSCode Version: 1.47.2 (user setup)
- OS Version: Windows_NT x64 10.0.19041

Steps to Reproduce:

1. Disable the `TypeScript and JavaScript Language Features` extension from the Extensions `@builtin`
2. Try to enable it 

**`@disabled `**

![image](https://user-images.githubusercontent.com/5789670/87882484-d9274500-ca08-11ea-9849-0857e792bf57.png)

**`@builtin`**

![image](https://user-images.githubusercontent.com/5789670/87882688-8a7aaa80-ca0a-11ea-8f91-e08bd36ba31f.png)

Does this issue occur when all extensions are disabled?: Yes

![image](https://user-images.githubusercontent.com/5789670/87882317-fdceed00-ca07-11ea-9ced-514820485627.png)

This issue occurs only with the project/folder that I have disabled for it. Rest isn't affected.

**Comment by seahindeniz** [comment]
I've found a temporary solution. I have created a symbolic link to the project folder to continue to work on the project and have the extension enabled.

But this doesn't change the original status of the main issue

**Comment by seahindeniz** [comment]
There is one possibility to reset workspace related configurations/states which helped me to reset extensions current state.

I have found this folder %userprofile%\AppData\Roaming\Code\User\workspaceStorage

As I understand, this folder contains workspace related databases.
All folders under `workspaceStorage` have generated names.
I have looked at every workspace.json files inside every folder to identify the folder that I want to reset. 
Once I found, I just simply deleted the folder named as `2882fa7d49d6d9d80b49cb04f5900284`.
After that, I have reopened my projects folder and there was no problem that I can report so far.

Of course, this approach allows you to have reset, not fix.
